website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -48,8 +50,40 @@ $ make lint
 
 **NOTE** Note that for it is required to install `golangci-lint` locally. For more info see its [doc](https://github.com/golangci/golangci-lint#install)
 
+
+## How the operator-sdk images are built
+
+Each time a Pull request is merged, a new image is built and pushed to the `quay.io/operator-framework` repository. This is done by the Travis deploy stage, which is configured in our [.travis.yml][travis].
+
+See that the images are built with different architectures from master as it is also done in the release process. See in the release Tab of the operator-sdk repository, that each version has different architectures as well for each supported operator type such as Helm and Ansible for example. 
+
+In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113) which will call the same targets which are used by the Travis in the master CI job to build these images. To know more about the release process, see the [doc][release-doc].
+
+## How to test the build of operator-sdk images
+
+If you are looking for to contribute to it, then you will need to ensure that the changes made in your Pull Request will also be executed successfully in the `Deploy` stage which, as described above, will just be trigged from the master branch.
+
+So, you will need to enable the Travis in your fork repository and perform a few configurations to allow it to be executed and check it. Following the steps. 
+
+- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
+- Set the following environment variables in the Travis settings for your fork:
+
+    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
+    - `HELM_IMAGE` same as above, but for helm
+    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
+    - `DOCKER_USERNAME` credentials for your repo
+    - `DOCKER_PASSWORD` credentials for your repo
+    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` 
+    - `COVERALLS_TOKEN`  token to integrate the project with `https://coveralls.io/`. So, enable your fork in and generate a token to allow it. 
+
+- Make a commit with `[travis deploy]` in the commit message on the branch with the aarch64 changes.
```

Comment:
Is this only if the branch has aarch64 changes, or any branch that we want to test the travis deploy?

Reply:
the aarch64 should not be in. Good spot.